sos/plugins/__init__.py

```diff
@@ -1570,6 +1574,15 @@ def setup_verify(self):
         if verify_cmd:
             self.add_cmd_output(verify_cmd)
 
+    def _postproc(self):
```

Comment:
The scoping looks funny here - normally the public (non `_`-prefixed) version of the method is the wrapper, and the private method is the wrapped one.

I think it would be less confusing to re-name the existing method to `_postproc()`, and have `sosreport` continue to call `plug.postproc()` - then depending on the option value the public method calls or doesn't call `self._postproc()` to do the actual work.

Reply:
The plugins doing postprocessing today all override `postproc()`, hence why I went with this approach. So if `sosreport` still calls that directly, we'd need to change the plugins to now override `_postproc()` instead, no? Seemed a bit awkward to me.

The other option could be to do this check directly in `sosreport`, e.g.:

```python
    def postproc(self):
        for plugname, plug in self.loaded_plugins:
            try:
                if plug.get_option('postproc'):
                    plug.postproc()
                else:
                    self.log_info('some skip message')
```

Reply:
Yeah, I see what you mean - although I think having the `get_option()` check in sosreport looks better - we aren't super strict about public vs. private in sos (and neither is Python ;-), but it looks weird to have a "private" method called by another module, that calls a similarly-named "public" method.